### ISSUE TITLE
Incorrect margin position for ylabel

### ISSUE THREAD

**Issue by blas-ko** [JuliaPlots/Plots.jl#2024]
Hello! 
I'm trying to make some plots with custom font sizes using math labelling using LaTexStrings. The problem is that in the output there's not a correct margin for the `y` label. 
Any idea what can be causing this?  I'm using standard `gr()` backend.

Here's an example code:  
```julia
using Plots, LaTeXStrings

T = 0:0.1:10
X = rand(length(T))

plot(T,X)
plot!(xlabel=L"T", ylabel=L"X(T)",
      leg=:false, guidefont=font(16), tickfont=font(12))
```
Here's my output:
![image](https://user-images.githubusercontent.com/8909006/57535280-c95a9e00-7339-11e9-9c70-d1fa9e5decc6.png)

Many thanks! 

<hr>

*EDIT: I can confirm is not a problem of `LaTexStrings` as I tried to label the axis with normal text and the same problem persists.*

**Comment by daschw** [comment]
This is already fixed on master (#2018), just not yet released. I'm on it.

**Comment by daschw** [comment]
This should be fixed with Plots v0.25.0.
